packages/storybook/stories/wc-helpers.jsx

```diff
@@ -234,16 +250,16 @@ export function StoryDocs({ componentName, data, children }) {
   const componentDocs = additionalDocs?.[component];
   const componentData = data ? { ...data, ...componentDocs } : null;
 
-  const maturityCategory = componentDocs?.maturityCategory;
-  const maturityLevel = componentDocs?.maturityLevel;
+  const maturityScale = getMaturityScale(data.docsTags);
+
+  const maturityCategory = maturityScale.category ?? componentDocs?.maturityCategory;
+  const maturityLevel = maturityScale.level ?? componentDocs?.maturityLevel;
   // This feels a bit awkward, but I didn't want to use a magic number
   const _componentName = data?.tag?.slice('va-'.length);
   // Default the guidance values to be based on the web component's
   // tag name where possible
-  const {
-    guidanceHref = _componentName,
-    guidanceName = capitalize(_componentName)?.replaceAll('-', ' '),
-  } = componentDocs;
+  const guidanceHref = componentDocs?._componentName;
```

Comment:
I haven't followed all the commits so I don't know if it's related to the recent `@maturityCategory` changes or not, but I'm not sure if this will do what we think it will.